src/components/ArticleList.jsx

```diff
@@ -17,12 +18,12 @@ export const ArticleList = () => {
             })
     }, [])
 
-    const authorName = articles
+    const authorNameReference = articles
```

Comment:
If you want to do this, please extract this out into a tested utils function.
You are exploiting implicity returns and the && operator, however this isn't improving the quality of the code produced.